Hashids/Hashids.swift

```diff
@@ -129,10 +129,11 @@ public class Hashids_<T: protocol<Equatable, UnsignedIntegerType>>: HashidsGener
     
     public func encode(values: [Int64]) -> String? {
         let ret = _encode(values)
-        return ret.reduce(String(), combine: { (var so, i) in
+        return ret.reduce(String(), combine: { (so, i) in
+            var mutableSo = so
```

Comment:
imo if we have no intention of using so outside of making it mutable, I like `var so = so` as it's more explicit about what you're trying to do.